### PR TITLE
[BL] Simple pvtu handling

### PR DESCRIPTION
**Pull request by jkozdon**

### CODE REVIEW

**Comment by simonbyrne** [comment]
Should this perhaps go somewhere else?

**Comment by jkozdon** [comment]
> Should this perhaps go somewhere else?

Maybe the existing `Mesh/vtk.jl` file? Right now this file is just included in `DGBalanceLawDiscretization.jl`. Probably not the best approach overall, though it works...

**Comment by simonbyrne** [comment]
Yes, cross-including files seems like a bad idea. Can we move the vtk/pvtu stuff to its own module?

**Comment by jkozdon** [comment]
That's fine with me. I'd also thought perhaps having the VTK functionality lives as functions of the Grid module. Given a grid and an MPIArray we can write VTK files. I just haven't had a chance to do this yet. Maybe for now I will add it to `vtk.jl`, and we can move stuff around later? Just thinking that anyone running in parallel will want pvtu sooner rather than later(?).

File: test/DGmethods/Euler/isentropic_vortex_standalone.jl [hunk 1]
```diff
@@ -175,7 +175,13 @@ function main(mpicomm, DFloat, topl::AbstractTopology{dim}, N, timeend,
     outprefix = @sprintf("vtk/isentropicvortex_%dD_mpirank%04d_step%04d",
                          dim, MPI.Comm_rank(mpicomm), step[1])
     @debug "doing VTK output" outprefix
-    writevtk(outprefix, Q, spacedisc, statenames)
+    DGBalanceLawDiscretizations.writevtk(outprefix, Q, spacedisc, statenames)
```
Comment (charleskawczynski) [inline test/DGmethods/Euler/isentropic_vortex_standalone.jl]: Can we remove `DGBalanceLawDiscretizations.`?

Comment (simonbyrne) [reply test/DGmethods/Euler/isentropic_vortex_standalone.jl]: yes, that should be removed

**Pull request merged**